nova/files/newton/nova-controller.conf.Debian

```diff
@@ -21,7 +21,6 @@ ram_allocation_ratio = {{ controller.ram_allocation_ratio }}
 disk_allocation_ratio = {{ controller.disk_allocation_ratio }}
 scheduler_default_filters = {{ controller.scheduler_default_filters }}
 scheduler_available_filters = nova.scheduler.filters.all_filters
-scheduler_available_filters = nova.scheduler.filters.pci_passthrough_filter.PciPassthroughFilter
```

Comment:
Because (at least in newton and ocata) it's redundant of the line above it, no reason to explicitly add it to the available filters because it's already there.

```
$ /work/venvs/nova-newton/bin/python -c "import nova.version, nova.scheduler.filters
print nova.version.version_string();
print '\n'.join(str(filter) for filter in nova.scheduler.filters.all_filters() if 'pci' in str(filter))
"
14.0.8
<class 'nova.scheduler.filters.pci_passthrough_filter.PciPassthroughFilter'>


$ /work/venvs/nova-ocata/bin/python -c "import nova.version, nova.scheduler.filters
   print nova.version.version_string()
   print '\n'.join(str(filter) for filter in nova.scheduler.filters.all_filters() if 'pci' in str(filter))
   "
15.0.6
<class 'nova.scheduler.filters.pci_passthrough_filter.PciPassthroughFilter'>

```